src/sql/workbench/contrib/executionPlan/browser/executionPlanView.ts

```diff
@@ -5,6 +5,8 @@
 
 import * as azdata from 'azdata';
 import * as DOM from 'vs/base/browser/dom';
+import * as os from 'os';
```

Comment:
You can't use os or path modules directly in core code, see https://github.com/microsoft/vscode/wiki/Source-Code-Organization#layers for more details.

Instead there are local wrappers for functionality, such as [join(...)](https://github.com/Microsoft/azuredatastudio/blob/26455e9113fb97398b2c86273af64c42d218a71a/src/vs/base/common/path.ts#L178) from vs/base/common/path

Reply:
Thanks, updated.